### PR TITLE
Add export of cc in markdown

### PR DESCRIPTION
**Pull request by kanekotic**
Hello @rubik I am adding this PR to add an extra output format as a markdown table. 
The intention is to be able to save and view the report in a format that is easy to integrate in a static site.

### CODE REVIEW

**Comment by kanekotic** [comment]
@rubik

File: docs/commandline.rst [hunk 1]
```diff
@@ -158,6 +158,11 @@ Options
    If given, the results will be converted into XML. Note that not all the
    information is kept. This is specifically targeted to Jenkin's plugin CCM.
 
+.. option:: --xml
```
Comment (rubik) [inline docs/commandline.rst:161]: Shouldn't this be `--md`?

**Review by rubik — CHANGES_REQUESTED** [review]

**Comment by rubik** [comment]
Hi Alvaro, thanks for the contribution! I think it's a good idea and I will merge it shortly. Can you just address the comment I added?

File: docs/commandline.rst (hunk 1 above)
Comment (kanekotic) [reply docs/commandline.rst]: yep... you are completely right. Fixing 😄

**Comment by rubik** [comment]
@kanekotic Thanks, merged!

**Pull request merged**